src/NetSparkle.Tests/SparkleUpdaterFixture.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using NetSparkleUpdater;
+using NetSparkleUpdater.AppCastHandlers;
+using NetSparkleUpdater.Interfaces;
+
+namespace NetSparkleUnitTests
+{
+    public class SparkleUpdaterFixture : IDisposable
+    {
+        public const string CollectionName = "xmlappcast-fixture";
+
+        public SparkleUpdater CreateUpdater(string xmlData, string installedVersion, IAppCastFilter filter = null)
+        {
+            SparkleUpdater updater = new SparkleUpdater("test-url", new AlwaysSucceedSignatureChecker());
+            updater.AppCastDataDownloader = new StringCastDataDownloader(xmlData);
+            updater.Configuration = new EmptyTestDataConfguration(new FakeTestDataAssemblyAccessor()
+            {
+                AssemblyCompany = "NetSparkle Test App",
+                AssemblyCopyright = "@ (C) Thinking",
+                AssemblyProduct = "Test",
+                AssemblyVersion = installedVersion
+            });
+
+            if (filter != null)
+            {
+                XMLAppCast cast = updater.AppCastHandler as XMLAppCast;
+                if(cast != null)
+                    cast.AppCastFilter = filter;
+            }
+
+            return updater;
+        }
+
+        public string GetSimpleXmlAppCastData()
+        {
+            var appCastData = @"
```

Comment:
These XML app casts should be moved to separate .xml file(s) since they're so long. (Shoulda done this for the tests I added in the past, too.) That'll make the test files easier to read.

Reply:
Now the XML files are loaded from disk.